src/main/resources/application.properties

```diff
@@ -1,9 +1,13 @@
+# Project
 spring.application.name=online-book-store
-server.port=8081
+# Database
 spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
 spring.datasource.username=root
 spring.datasource.password=1234
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+# JPA
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
+# Liquibase
+spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.yaml
```

Comment:
looks like the default value. Do you really need it?

Reply:
Yes, you're right. It looks like the default value. I added it after a failed attempt to automatically fill my tables with data, but it didn't help, so I'm removing it.